users/serializers_test.py

```diff
@@ -96,6 +96,17 @@ def test_validate_postal_code_formats(sample_address, data, error):
     assert str(serializer.errors["postal_code"][0]) == error
 
 
+@pytest.mark.parametrize("postal_code", ["K0M 2K0", "J8R 3P7", "G3G 2B7", "L6L 1C2"])
```

Comment:
any postal code test with `-` ?

Reply:
I don't think that CA postal codes exists with `-`. Ref https://www.postalcodesincanada.com/

Reply:
I think, you removed from the corresponding regex for `-` in the new commit. I saw there so asked.

Reply:
yes, I realized and removed.